Framework/Source/Effects/AmbientOcclusion/SSAO.cpp

```diff
@@ -240,10 +240,10 @@ namespace Falcor
 
     void SSAO::reflect(RenderPassReflection& reflector) const
     {
-        reflector.addInput(kColorIn);
-        reflector.addOutput(kColorOut);
-        reflector.addInput(kDepth);
-        reflector.addInput(kNormals).setFlags(RenderPassReflection::Field::Flags::Optional);
+        reflector.addInput(kColorIn).setSampleCount(1);
```

Comment:
I wonder if the default for sample-count should be 1. Then if someone doesn't care they should set it to 0 (or -1)